vmware-event-router/deploy/event-router-config-knative.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: event-router.vmware.com/v1alpha1
+kind: RouterConfig
+metadata:
+  name: router-config-knative
+  labels:
+    key: value
+eventProvider:
+  type: vcenter
+  name: veba-demo-vc-01
+  vcenter:
+    address: https://my-vcenter01.domain.local/sdk
+    insecureSSL: false
+    checkpoint: false
+    auth:
+      type: basic_auth
+      basicAuth:
+        username: administrator@vsphere.local
+        password: ReplaceMe
+eventProcessor:
+  type: knative
+  name: veba-demo-knative
+  knative:
+    address: http://broker-ingress.knative-eventing.svc.cluster.local/default/default
```

Comment:
So if you use the duck here, you can then be more flexible, like targeting a Serving Service, Sequence, etc. and not have to have the user have to specify the full URL if they do not want to and it makes it more reliable to provide templates, etc. that work on clusters where for example the cluster dns does not end in cluster.local and so forth.